Qa.py

```diff
@@ -29,47 +29,39 @@ def get_quarter_total(cur):
     return float(cur.fetchone()[0])
 
 def get_count_student_attempts_n_units_for_quarter(cur, units):
-    cur.execute("SELECT COUNT(sid) "\
-                "FROM "\
+    cur.execute("SELECT COUNT(S.sid) "\
+                "FROM Student S, "\
```

Comment:
The cross product with S above results with a different count if it is not used here.